src/components/file/file.repository.ts

```diff
@@ -194,9 +196,11 @@ export class FileRepository {
         .with('*')
         .optionalMatch([
           node('requestingUser'),
-          relation('in', '', 'member'),
-          node('acl', 'ACL', { [camelCase(`canRead-${prop}`)]: true }),
-          relation('out', '', 'toNode'),
+          relation('in', '', 'member*1..'),
```

Comment:
@michaelmarshall just dropped this with #1193 right?
```suggestion
          relation('in', '', 'member'),
```

Reply:
removed